portfolio/src/main/webapp/index.html

```diff
@@ -5,17 +5,17 @@
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css" />
     <link rel="stylesheet" href="/buttons.css" />
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyC3oaiqJFYwy7VaT34N7icYTwjAFXu-umY"></script>
     <script src="script.js"></script>
   </head>
-  <body onload="getDataServlet()">
+  <body onload="initMap()">
     <div id="content">
       <h1>Brett Allen's Portfolio</h1>
       <h3>
         <a href="https://www.linkedin.com/in/brett-allen-635aba171/" target="_blank" class="button button-blue"
           >LinkedIn</a
         >
         <a href="https://github.com/brettallenyo" target="_blank" class="button button-black">GitHub</a>
-        <a href="/infiniterun/gamepage.html" class="button button-yellow">Play Infinite Run!</a>
         <a href="/theoffice/theoffice.html" class="button button-purple">Office Quotes</a>
         <a href="/volleyball/volleyball.html" class="button button-red">Volleyball</a>
       </h3>
```

Comment:
not relevant to this PR but you probably want to add some margin to your `description` div below so the words don't overflow to the sides of the page

Reply:
Ok I will in the next, thank you